src/VisualStudio/Core/Def/TaskList/VisualStudioTaskListService.cs

```diff
@@ -93,6 +105,35 @@ private async Task StartAsync(Workspace workspace)
             }
         }
 
+        private async Task WaitUntilTaskListActivatedAsync()
```

Comment:
from Dabid Pugh, this is the correct way to both determine if hte task-list is actually visible, and register to find out if it is visible if not.